### PR TITLE
Fix #7098

### LINKED ISSUE #7098: [PAGE ISSUE]: 'Using Actions and Shortcuts'
### Page URL

https://docs.flutter.dev/development/ui/advanced/actions_and_shortcuts/

### Page source

https://github.com/flutter/website/tree/main/src/development/ui/advanced/actions_and_shortcuts.md

### Describe the problem

The article mentions the `CallbackShortcuts` is "(available on the dev branch)" which is no longer true as this is already on the stable branch, at least from 2.10 

### Expected fix

Remove parenthesized text

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by fabiancrx**
Feature already landed stable

**IMPORTANT:** Due to work on the docs.flutter.dev infrastructure, this repo is **not accepting pull requests**.

Instead of creating a PR, please file an issue (https://github.com/flutter/website/issues/new/choose) about the needed change.

We expect to start accepting PRs again the week of May 11.

### CODE REVIEW

**Review by sfshaza2 — COMMENTED** [review]
@fabiancrx, hi! We aren't accepting PRs at the moment, though your edit IS a good one! We can revisit after the next stable release. I'm closing for now.

**Comment by domesticmouse** [comment]
Please update this PR to bring it up to `main`.

**Comment by fabiancrx** [comment]
Just rebased right now @domesticmouse

File: src/development/ui/advanced/actions_and_shortcuts.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ bindings, and the implementation of those bindings, are in different places.
 If indeed all that is needed is a callback, without all the complexity (or
 flexibility) of `Actions` and `Shortcuts`, you can already use a `Focus` widget
 for this. For example, here's the implementation of Flutter's simple
-[`CallbackShortcuts`][] widget (available on the dev branch) that takes a map of
+[`CallbackShortcuts`][] widget that takes a map of
```
Comment (domesticmouse) [inline src/development/ui/advanced/actions_and_shortcuts.md:82]: Please merge the following line onto this line to fill things out to 80 chars.

**Review by domesticmouse — APPROVED** [review]

File: src/development/ui/advanced/actions_and_shortcuts.md (hunk 1 above)
Comment (fabiancrx) [reply src/development/ui/advanced/actions_and_shortcuts.md]: Done!

**Review by domesticmouse — APPROVED** [review]

**Pull request merged**